toonz/sources/toonz/startuppopup.cpp

```diff
@@ -346,16 +360,21 @@ void StartupPopup::showEvent(QShowEvent *) {
 
   // update recent scenes
   // clear items if they exist first
+  refreshRecentScenes();
+  m_recentSceneLay->addStretch(100);
```

Comment:
Adding the stretch should be moved at the end of `refreshRecentScenes()` .
If you just would like to put the items aligned at the top, it is enough to add stretch with the stretch value 1 and set the stretch value of the "scene name" items to 0 like as follows:
 
<img src="https://cloud.githubusercontent.com/assets/17974955/23702034/e57675aa-043c-11e7-89ec-59e19118d174.png" width=400>